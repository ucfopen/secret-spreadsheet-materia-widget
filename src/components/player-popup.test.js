import React from 'react'
import Popup from './player-popup'
import renderer from 'react-test-renderer';

describe('Popup component', () => {
	beforeEach(() => {
		jest.resetModules();
	});

	test('Is rendered', () => {
		const props = {
			handlePopupToggle: jest.fn()
		}

		const component = renderer.create(<Popup {... props} />);
		const tree = component.toJSON();

		expect(tree).toMatchSnapshot();
	});
});
