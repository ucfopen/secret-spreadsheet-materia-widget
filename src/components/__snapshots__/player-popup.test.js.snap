// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Popup component Is rendered 1`] = `
<div
  className="popup help"
>
  <div>
    <h2>
      Secret Spreadsheet - How it works:
    </h2>
    <div>
      <img
        alt="Secret Spreadsheet Instructions"
        src="assets/img/player-instruction-placeholder.jpg"
      />
      <p>
        Cells of this spreadsheet have been 
        <strong>
          intentionally hidden.
        </strong>
      </p>
      <p>
        Input the hidden data correctly 
        <strong>
          (capitalization matters!)
        </strong>
         to restore the spreadsheet.
      </p>
    </div>
    <button
      onClick={[MockFunction]}
      type="button"
      value="Got it!"
    >
      Got it!
    </button>
  </div>
</div>
`;
